### ISSUE TITLE
Accessibility issue on embedded board when using link and target is not separate page (the table settings)

### ISSUE THREAD

**Issue by peho72** [brickfield/moodle-mod_board#166]
Hi,
In Moodle 4.1 Board plugin newest release 1.401.05 (added: also same issue in previous versions):
When you create board and show it embedded on the course main page and open note and click the link there the link page open to the embedded frame. 
The page content will be on that small frame, not whole page. e.g. if the target page is big image, you can see only part of the image/table. Zoom in/out will only change the size of the frame / current Moodle page, not the target page. 
Added: Tested on some links like ["pinterest"](https://fi.pinterest.com/), embedded didn't show nothing but gray page.

When you change the links target to the separate page, there is no other accessibility issue but opening the new tab / window to the browser (not so big issue than not see all information on the target page).

If this cannot be fixed properly, please add at least the notification to the **tooltip** of this accessibility issue.
Settings > Board settings
   [ ] Enable blank target (?) <- **tooltip**

When the board is not embedded (or opened to full size), the target page open full size over current page (or if link open to separate plank tab/window then there).

Thanks for noticing this.
And greetings from Turku.

**Comment by learningtechnologyservices** [comment]
Hi there @peho72 ! Many thanks for reporting this situation, and we will look into it shortly! And greetings from Ireland!

**Comment by learningtechnologyservices** [comment]
Hi @peho72 , can we propose the following update to the board instance setting for enableblanktarget:

Underneath, or inside the tooltip, it should include the following:
"This setting is ONLY recommended for when the board instance is embedded into the course page. If NOT embedded, this should be disabled, otherwise any links in posts will open in new tabs in the browser."

Would this be be clear for your users? Thanks in advance.

**Comment by peho72** [comment]
Hi,
If it is directly instructed with tool tips in the board activity settings, then your proposal is good idea. 
Anything that is possible but not prevent to do in certain situation (even if recommended not to), teachers do it because they don’t have time to read instructions pages like in Moodle Docs.
This is just what I happen to know with over 20 years experience in IT support. They didn’t read much instructions back in 90’s when they had more time and definitely not now, when they don’t have time unless directly pointed that here in this instructions is written what to do and why in particular way, not in the way they want or is intuitive.

Summa summarum: Good tool tips may lead good results.

Petri Hörkkö

**Comment by learningtechnologyservices** [comment]
Hi @peho72  , this issue has now been resolved in our latest tag 1.405.06, here and in the Marketplace directory.

Thanks once again for your detailed feedback and for using Board!
